CHANGELOG.md

```diff
@@ -1,3 +1,41 @@
+# 107.1.0
+
+This minor release we added support to Snackbar to inform users when it has disappeared. As well as, improvements around touch targets for Chips, alignment improvements to Dialogs and various other bug improvements.
+
+## New features
+
+In this release we added a new method to the `MDCSnackbarManagerDelegate` protocol to inform users when the snackbar disappeared.
```

Comment:
In this release we added a new method to the `MDCSnackbarManagerDelegate` protocol that informs the user when a snackbar disappears.

Reply:
Done